### ISSUE TITLE
Problem loading module in IE 11

### ISSUE THREAD

**Issue by classifaddict** [QQuick/Transcrypt#634]
Hello,
Here is how I initialize my app:
```
<html>
<head>...</head>
<body>
...
    <script type="module">import * as mymodule from "/myapp/static/__target__/mymodule.js"; window.mymodule = mymodule;</script>
    <script>
        $(function(){
            MYMODULE = mymodule.Init();
        });
</script></body></html>
```

This is working fine for Chrome 73, FF 66 and Edge 41 but not for IE 11 who returns this error:
SCRIPT5009: 'mymodule' is undefined
jquery-3.4.1.min.js (2, 31118)

Is this a bug in Transcrypt 3.7.16 or is there a solution for IE 11?

Thanks,
Olivier

**Comment by classifaddict** [comment]
When I try this in IE 11 web dev console:
import * as mymodule from "/myapp/static/__target__/mymodule.js";
I get:
Syntax error

**Comment by classifaddict** [comment]
So, it seems that IE 11 does not support ES 6 import statement.
If I compile with -e 5, how do I need to call my app then?

**Comment by faerot** [comment]
https://github.com/QQuick/Transcrypt/issues/563#issuecomment-438272825

**Comment by classifaddict** [comment]
Thanks @faerot  for this reference which confirms the fact that ES5 is not supported by TS 3.7.
The only workaround I found so far is to use a separate Python 3.6.8 env with TS 3.6.101 installed to compile my code (luckily I don't have to adapt the source Python code to make it work) but I have to remember to avoid 3.7+ features from now. 
Hopefully I will able to drop IE 11 support in a couple of years and come back to a recent TS version.

**Comment by faerot** [comment]
> 
> 
> Thanks @faerot for this reference which confirms the fact that ES5 is not supported by TS 3.7.
> The only workaround I found so far is to use a separate Python 3.6.8 env with TS 3.6.101 installed to compile my code (luckily I don't have to adapt the source Python code to make it work) but I have to remember to avoid 3.7+ features from now.
> Hopefully I will able to drop IE 11 support in a couple of years and come back to a recent TS version.

I was using 3.6.101 for quite a while for the same reason, but it has way too many bugs unfixed. Its better to install RollupJS and other javascript tools which will merge and transpile ES6 to ES5 instead

**Comment by classifaddict** [comment]
Do you know where I can find which bugs have been fixed since 3.6.101?
Transcrypt documentation recommends usage of ParcelJS... I guess that this tool is able to transpile to ES5 as well. If this is the case, the documentation should mention that it could also be the solution to support ES5 compliant platforms such as IE 11 and make it clear that TS 3.7 does not support ES5.
Anyway, I know how hard is to maintain such a project... Thanks  a lot for that Jacques!

**Comment by JdeH** [comment]
The decision to drop support for ES5 was a hard one, but it slowed down new development too much. The idea was to keep available old versions of Transcrypt for that, but indeed they have more bugs that weren't yet fixed.
There are some ES5 bits left, which is confusing. They will be weed out gradually.
